_schema.graphql

```diff
@@ -2006,6 +2006,230 @@ type BuyersPremium {
   percent: Float
 }
 
+type BuyOrder implements Order {
+  # ID of the order
+  id: ID
+
+  # Order Mode
+  mode: OrderModeEnum
+
+  # Currency code of this order
+  currencyCode: String
+
+  # State of the order
+  state: String
+
+  # Reason for current state
+  stateReason: String
+
+  # Tracking code of the order
+  code: String
+
+  # Order Requested Fulfillment
+  requestedFulfillment: RequestedFulfillment
+
+  # Latest offer
+  lastOffer: Offer
```

Comment:
If I understand correctly, offer-specific fields will be removed eventually from `Order`interface and `BuyOrder` type, right?

Reply:
yes, `lastOffer` and `offers` will be deprecated